clusterman/aws/aws_resource_group.py

```diff
@@ -94,6 +97,18 @@ def get_instance_metadatas(self, state_filter: Optional[Collection[str]] = None)
             instance_metadatas.append(metadata)
         return instance_metadatas
 
+    def _is_instance_cordoned(self, instance_dict: InstanceDict) -> bool:
+        # If EC2 node has no tags, None is returned.
+        # This sets the value to [] instead.
+        tags = instance_dict.get("Tags", [])
+
+        for tag in tags:
+            if tag["Key"] == "cordon":
+                if tag["Value"] == "true":
+                    return True
+
+        return False
+
```

Comment:
technically this doesn't need to be inside this class since it doesn't use any instance attributes - but that's pretty nitpicky so feel free to ignore

Reply:
That's a good call. I added the function to the class because it's relevant to it (note the leading `_` for example), although it never really uses `self` indeed. It's more of a design and relevancy choice rather than a technical requirement one. I'd rather keep things ordered that way, so I'm going to merge.